tests/unit/db/result_test.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+# Copyright (c) 2015 IBM. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+result module unit tests
+"""
+import unittest
+
+from cloudant.design_document import DesignDocument
+from cloudant.errors import CloudantArgumentError
+from cloudant.result import python_to_couch, Result
+from cloudant.views import View
+from tests.unit.db.unit_t_db_base import UnitTestDbBase
+
+
+class PythonToCouchTests(unittest.TestCase):
+    """
+    test cases for validating python_to_couch's options
+    """
+
+    def test_valid_option_group_level(self):
+        """
+        Test group_level option is valid
+        """
+        opts = {
+            "group_level": 100
+        }
+        result = python_to_couch(opts)
+        self.assertEqual(result['group_level'], 100)
+
+    def test_invalid_option_group_level(self):
+        """
+        Test group_level options are invalid
+        """
+        opts = {
+            "group_level": True
+        }
+        self.assertRaises(CloudantArgumentError, python_to_couch, opts)
+        opts = {
+            "group_level": "100"
+        }
+        self.assertRaises(CloudantArgumentError, python_to_couch, opts)
+
+
+class ResultTests(UnitTestDbBase):
+    """
+    tests for Result class
+    """
+
+    def setUp(self):
+        """
+        Set up test attributes
+        """
+        super(ResultTests, self).setUp()
+        self.db_set_up()
+        self.ddoc = DesignDocument(self.db, 'ddoc001')
+        self.view = View(
+                self.ddoc,
+                'view001',
+                'function (doc) {\n  emit(doc._id, 1);\n}',
+                '_count'
```

Comment:
I think these lines are double indented.  There should only be indented 4 spaces.  Looks like they are indented 8 spaces.